backend/sample.env

```
@@ -111,3 +111,6 @@ CACHE_TTL_SEC=10800
 # Default auth credentials
 DEFAULT_AUTH_USERNAME=
 DEFAULT_AUTH_PASSWORD=
+
+# Default log level
+DEFAULT_LOG_LEVEL="DEBUG"
```

@kirtimanmishrazipstack the default value can be INFO itself here
```suggestion
DEFAULT_LOG_LEVEL="INFO"
```

---

Why @chandrasekharan-zipstack ? In unstract-cloud logs we want to view debug logs right ?

---

done